blog/content/post/2023-04-27-3mdeb-open-source-contribution.md

```diff
@@ -24,12 +28,17 @@ categories:
 
 ---
 
-We are excited to share with you the open-source contributions of our team.
-We have been actively contributing to a number of open-source projects that have
-a significant impact on the industry. Through these contributions, we have been
-able to help improve the overall functionality and security of these projects,
-while also gaining valuable experience and knowledge that we can apply to our
-client work.
+From 2021 to 2022, our team of developers thrived, making significant
+contributions to firmware projects. Our primary focus revolved around
+[coreboot](https://3mdeb.com/training/#coreboot-for-embedded-linux-developers),
+a firmware solution that captured our attention. We also dedicated efforts to
+the advancement of fwupd and flashrom firmware update systems. But our
```

Comment:
Why not link to the fwupd and flashrom project pages, if we don't have associated tags in blog